tensorboard/webapp/metrics/data_source/metrics_data_source_test.ts

```diff
@@ -28,15 +34,23 @@ import {MetricsDataSource, PluginType} from './types';
 describe('TBMetricsDataSource test', () => {
   let httpMock: HttpTestingController;
   let dataSource: MetricsDataSource;
+  let store: MockStore<State>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       imports: [TBHttpClientTestingModule],
-      providers: [{provide: MetricsDataSource, useClass: TBMetricsDataSource}],
+      providers: [
+        {provide: MetricsDataSource, useClass: TBMetricsDataSource},
+        provideMockStore(),
+      ],
     }).compileComponents();
 
     httpMock = TestBed.inject(HttpTestingController);
     dataSource = TestBed.inject(MetricsDataSource);
+    store = TestBed.inject<Store<State>>(Store) as MockStore<State>;
+    store.overrideSelector(selectors.getFeatureFlags, buildFeatureFlag());
```

Comment:
Do you need to override this selector given that you've overriden the two selectors you are directly using?